code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -768,6 +768,16 @@
 	name = "Ladies Commie"
 	icon_state = "female_commie"
 	gender = FEMALE
+	
+/datum/sprite_accessory/underwear/sports_bra

```

Comment:
So they have a bra.... where are the actual underwear though? Having it as sprite_accessory/underwear would mean that they're half-nude since you don't have any underwear in the sprite.